deployment/permissions/partner.-24.ini

```diff
@@ -10,4 +10,9 @@ permission2.type = 4
 permission2.name = PARTNER_-24_GROUP_*_PERMISSION
 permission2.friendlyName = "Partner -24 permission for group *"
 permission2.description = "Partner -24 permission for group *"
-permission2.partnerGroup = *
\ No newline at end of file
+permission2.partnerGroup = *
+
+permission3.partnerId = -24
```

Comment:
why you need this permission?
for resetJobCatalogItem you added -24>PARTNER_-24_GROUP_*_PERMISSION